eng/devices/ios.cake

```diff
@@ -6,13 +6,14 @@
 #tool nuget:?package=NUnit.ConsoleRunner&version=3.16.3
 
 const string defaultVersion = "16.4";
-const string dotnetVersion = "net7.0";
+const string dotnetVersion = "net8.0";
```

Comment:
This is in several `*.cake` files. Can we put it in a common/shared file and declare it once?